internal/indexer/definitions/torrentseeds.yaml

```diff
@@ -0,0 +1,65 @@
+---
+#id: TorrentSeeds
+name: TorrentSeeds
+identifier: torrentseeds
+description: TorrentSeeds (TS) is a GENERAL/0-DAY tracker with great pretimes.
+language: en-us
+urls:
+  - https://torrentseeds.org
+privacy: private
+protocol: torrent
+supports:
+  - irc
+  - rss
+source: custom
+settings:
+  - name: passkey
+    type: secret
+    label: Passkey
+    help: "Go to General / Get RSS / Check a category and click create your passkey is after torrent_pass="
+
+irc:
+  network: Torrentseeds.org
+  server: irc.torrentseeds.org
+  port: 6697
+  tls: true
+  channels:
+    - "#announce"
+  announcers:
+    - torrentseeds
+  settings:
+    - name: nickserv.account
+      type: text
+      required: false
+      label: NickServ Account
+      help: NickServ account. Make sure to group your user and bot. Eg. user_bot
+    - name: nickserv.password
+      type: secret
+      required: false
+      label: NickServ Password
+      help: NickServ password
+    - name: invite_command
+      type: secret
+      default: "Cerberus identify username irckey"
+      required: true
+      label: Invite command
+      help: Invite auth with Cerberus.
+
+parse:
+  type: single
+  lines:
+    - test:
+        - "New: This.Is.A.New.show.S00E00.720p.WEB.H264-Test .:. Category: TV/HD .:. Size: 364.82 MB .:. URL: https://www.torrentseeds.org/details.php?id=000000 .:. Uploaded by: George."
+      pattern: 'New: (.*) .:. Category: (.*) .:. Size: (.*) .:. URL: (https?\:\/\/[^\/]+)\/.*id=(\d+) .:. Uploaded by: (.*).'

```

Comment:
Looking at the actual announces and in the autodl-irssi tracker files, they actually have 2 spaces after `URL:`.

`- "New: This.Is.A.New.show.S00E00.720p.WEB.H264-Test .:. Category: TV/HD .:. Size: 364.82 MB .:. URL:  https://www.torrentseeds.org/details.php?id=000000 .:. Uploaded by: George."`

So fix the test by just adding a space, and same with the pattern, like: 

`New: (.*) .:. Category: (.*) .:. Size: (.*) .:. URL:  (https?\:\/\/[^\/]+)\/.*id=(\d+) .:. Uploaded by: (.*).`

Reply:
Ok I confirm this is working now
For me TS, MTV, Milkie and TD are working
I will work on TBy